taccsite_cms/contrib/taccsite_sample/tests.py

```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase
+from django.test.client import RequestFactory
+
+from django.contrib.auth.models import AnonymousUser, User
+
+from cms.api import add_plugin
+from cms.models import Placeholder
+from cms.plugin_rendering import ContentRenderer
+
+from taccsite_cms.contrib.taccsite_sample.cms_plugins import TaccsiteSamplePlugin
+from taccsite_cms.contrib.taccsite_sample.models import TaccsiteSample
+
+class TaccsiteSampleTests(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.auth_user = User()
+        self.anon_user = AnonymousUser()
+        self.context = { 'request': self.factory.get('/test/user') }
+        self.placeholder = Placeholder.objects.create(slot='test')
+        self.plugin = add_plugin(
+            self.placeholder,
+            TaccsiteSamplePlugin,
+            'en',
+        )
+
+    # Helpers
+
+    def _get_context(self):
+        """Return context necessary for testing plugin logic"""
+        plugin_instance = self.plugin.get_plugin_class_instance()
+        context = plugin_instance.render(self.context, self.plugin, None)
+        return context
+
+    def _get_html(self):
+        """Return markup necessary for testing plugin template"""
+        renderer = ContentRenderer(request=self.factory)
+        html = renderer.render_plugin(self.plugin, self.context)
+        return html
```

Comment:
__What If:__ If tests become standard, then this is boilerplate. But if plugins are moved into their own repo, then sharing this boiler plate may require more overhead.